.editorconfig

```diff
@@ -1,2 +1,24 @@
-[*.js]
-indent_size=2
+# editorconfig.org
+# editorconfig is a unified configuration file that all editors can take
+# into account
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.coffee]
+indent_size = 4
```

Comment:
That's a good idea to have a more general configuration (that's configure the editor to manage `.json`).
Very great work!
But why do you copy/paste some irrelevent `.editorconfig` instead of using a simple more accurate one?